src/slidem-deck.js

```diff
@@ -14,9 +14,9 @@ document.adoptedStyleSheets = [...document.adoptedStyleSheets, globalStyle];
  * non-capturing group (_optional_):
  *   `/step-`
  *   named capture group 2 `step`:
- *     Either **0-9** (_>= 1x_) or `Infinity`
```

Comment:
Infitity is a serializable number in js

Reply:
I understand, but is that something we want to support? Can we have a state where we are at the Infinity-th step? If so, why not also Infinity slides?

From the perspective of typing I see why you need to parse for Infinity here, but from a product perspective I don't see the use of that. The API can accept a non-Infinity number only, which makes more sense considering the product.

Reply:
please see 8a7cc585